pkg/occlient/occlient.go

```diff
@@ -2212,12 +2212,13 @@ func (c *Client) CreateServiceInstance(serviceName string, serviceType string, s
 
 // CreateServiceBinding creates a ServiceBinding (essentially a secret) within the namespace of the
 // service instance created using the service's parameters.
-func (c *Client) CreateServiceBinding(bindingName string, namespace string) error {
+func (c *Client) CreateServiceBinding(bindingName string, namespace string, labels map[string]string) error {
 	_, err := c.serviceCatalogClient.ServiceBindings(namespace).Create(
 		&scv1beta1.ServiceBinding{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      bindingName,
 				Namespace: namespace,
+				Labels:    labels,
 			},
 			Spec: scv1beta1.ServiceBindingSpec{
 				//ExternalID: UUID,
```

Comment:
IMO this particular line is having no significance over here, So it can be removed.

Reply:
It's not having a major significance. I've added it so that a service binding has a label for the app. It will help us have same labels for a service instance and service bindings. Hence, I'd prefer to keep it.

Reply:
she is talking about this commented like `//ExternalID: UUID,`

Reply:
Done